common/test/db_cache/bok_choy_schema_student_module_history.sql

```diff
@@ -34,7 +18,7 @@ CREATE TABLE `django_migrations` (
   `name` varchar(255) NOT NULL,
   `applied` datetime(6) NOT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=InnoDB AUTO_INCREMENT=119 DEFAULT CHARSET=utf8;
+) ENGINE=InnoDB AUTO_INCREMENT=88 DEFAULT CHARSET=utf8;
```

Comment:
This represents the next ID that will be used by MySQL for rows that are subsequently INSERTed without specifying a value for their primary key.

@Kelketek Do we need to include changes to the bok choy `db_cache` in this PR? As far as I know, normally that is left alone, and every one in a while the cache is updated as a separate PR. (Though sometimes it is necessary to rebuild it if it's causing test failures)